src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -349,6 +357,40 @@ class AudiusAPIClient {
     return adapted
   }
 
+  async getFavoritedTracks({
+    profileUserId,
+    currentUserId,
+    limit,
+    offset
+  }: GetProfileListArgs) {
+    this._assertInitialized()
+    const encodedUserId = encodeHashId(currentUserId)
+    const encodedProfileUserId = encodeHashId(profileUserId)
+    if (!encodedProfileUserId) {
+      throw new Error(`Unable to encode profile user id: ${profileUserId}`)
+    }
+    const params = {
+      user_id: encodedUserId || undefined,
+      limit,
+      offset
+    }
+
+    const endpoint = this._constructUrl(
+      ENDPOINT_MAP.favoritedTracks(encodedProfileUserId),
+      params
+    )
+    const favoritedTrackResponse: APIResponse<
+      { timestamp: string; item_type: 'track'; item: APITrack }[]
+    > = await this._getResponse(endpoint)
+    const adapted = favoritedTrackResponse.data
+      .map(({ item, ...props }) => ({
+        timestamp: props.timestamp,
+        track: adapter.makeTrack(item)
+      }))
```

Comment:
perhaps we should pull the adapter out into a makeFavoritedTracks. wdyt?

Reply:
actually, feel free to just merge. ill do this in my PR because im adding an activity type